guide/deployment/docker/build-a-dockerfile.md

```diff
@@ -0,0 +1,89 @@
+# Building your Docker image
+
+<Definition content="A Docker image is a read-only template containing instructions for creating a container that can run on the Docker platform. It provides an easy way to package up applications and pre-configured server environments that you can use privately or publicly with other Docker users." />
+
+Now that you have your `Dockerfile` and `.dockerignore` file, you can build your Docker image, with your instruction manual you made for the cli. Most people like to [automate this](#automating-the-build-process-with-github-actions) but you can also do it [manually](#manually-building-and-publishing-your-docker-image).
+
+## Manually building and publishing your Docker image
+
+Step 1: Build your Docker image
+
+```bash
+    docker build -t [your-image-name] .
+```
+
+Step 2: Push your Docker image to Docker Hub
+
+```bash
+    docker push [your-image-name]
+```
+
+## Automating the build process with GitHub Actions
+
+<Definition content="GitHub Actions is a CI/CD platform that allows you to automate your build, test, and deployment pipelines. You can set up workflows to build and test each pull request that comes into your repository, for this example we will be using GitHub Actions to publish a Docker Image every push you do to your master branch." />
+
+To use GitHub Actions your code must be in a GitHub repository.
+You can store your code in a git server such as GitHub for ease of access.
+
+### GitHub Desktop
+
+> A good choice for people who are scared of the terminal and don't wanna learn it, or simply just get things done quicker with a app.
+
+1. Download [GitHub Desktop](https://desktop.github.com/)
+2. [Create a repository](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/creating-your-first-repository-using-github-desktop)
+
+### Terminal
+
+> The classic way
+
+1. [Make sure git is installed](https://github.com/git-guides/install-git)
+2. [Create a repository](https://docs.github.com/en/get-started/quickstart/create-a-repo)
```

Comment:
This section could just be replaced with "Create a repository on GitHub", the guide doesn't needs to go this deep into make a single repository with cli and github desktop, simply linking to the docs might be better

Reply:
https://github.com/discordjs/guide/pull/1047/commits/1f036c57ff23940a5249a9bad266c8a9de65b83a